task-launcher/src/tasks/shared/helpers/randomizeStimulusBlocks.js

```diff
@@ -4,7 +4,7 @@ export function shuffleStimulusTrials(trialArray) {
   // 1. Group stimulus trials by type
   const stimulusTrialsByType = {};
   for (const trial of trialArray) {
-    if (trial.trialType !== ' instructions' && trial.notes !== 'practice') {
```

Comment:
I'm not sure why there was a space in this string, taking it out didn't seem to affect the math game.